nuxt.config.js

```diff
@@ -3,24 +3,6 @@ import fs from 'fs';
 export default {
   // Target: https://go.nuxtjs.dev/config-target
   target: 'static',
-
-  render: {
```

Comment:
Metamask + Firefox is broken with CSP when served via header (see https://github.com/MetaMask/metamask-extension/issues/3133#issuecomment-759116472). This doesn't affect our users because we only serve via header under local development -- but it does break local development. 

We serve production the CSP header is set by the meta tag here: https://github.com/thousandetherhomepage/ketherhomepage/blob/81cf90355e2c75b37cefc81c3f018365f1772b67/nuxt.config.js#L17

I think we can safely remove this, will confirm we are still doing CSP in the netlify preview

Reply:
Useful for Chrome dev